cmd/connector-gen/github.go

```diff
@@ -0,0 +1,114 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"net/netip"
+	"slices"
+	"strings"
+
+	"go4.org/netipx"
+)
+
+type GithubMeta struct {
```

Comment:
I'm assuming Github doesn't have a similar kb link as AWS, but just flagging incase it's forgotten!

Reply:
added!